packages/app-bridge-theme/src/types/Guideline.ts

```diff
@@ -56,53 +63,59 @@ export interface GuidelineDocumentLink {
     parentId(): Nullable<number>;
     type: 'document-link';
 }
+export interface DocumentLinkNavigationItem extends DocumentLinkBase {}
 
-export interface GuidelinePageCategory {
+interface PageCategoryBase {
     id(): number;
     title(language?: string): string;
     slug(language?: string): string;
-    children(): (GuidelineDocumentPage | GuidelineDocumentPageLink)[];
+    children(): (DocumentPageNavigationItem | DocumentPageHeadingNavigationItem)[];
     type: 'page-category';
 }
+export interface PageCategoryNavigationItem extends PageCategoryBase {}
```

Comment:
IMO we don’t need the `PageCategoryBase` for now. Once we need a `PageCategory` we can easily add it. Wdyt?

Reply:
same thing than for `documentGroup`